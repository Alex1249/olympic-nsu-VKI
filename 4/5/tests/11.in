22 62
-602 -870 -393 190 -135 638 -379 407 -856 -342 232 -205 730 377 -974 484 238 -515 23 -666 736 598
0 459
1 443
1 537
0 447
1 195
1 319
0 606
1 490
1 367
0 635
1 439
1 178
1 16
1 452
1 441
1 466
0 559
1 664
1 445
0 359
0 404
0 533
0 30
1 627
1 144
1 63
0 703
0 283
0 628
0 702
0 130
1 298
1 535
0 221
0 210
0 204
1 660
1 461
0 538
1 84
0 266
0 224
1 267
1 566
1 218
0 591
0 347
1 142
0 172
1 372
0 128
1 343
1 619
0 272
1 313
0 629
0 516
1 583
0 390
0 300
1 171
0 31
