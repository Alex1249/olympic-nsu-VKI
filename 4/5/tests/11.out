-604 -869 -393 190 1610612601 1057406 -532831 407 -2130776 -524565 488 -16777421 730 380 -1049550 2097636 462 -65539 16777237 -3738 2784 596 
